app/models/concerns/profile_searching.rb

```diff
@@ -9,6 +9,7 @@ module ProfileSearching
                          external parent_profile_id],
                   only_explicit: true
     scoped_search on: %i[name ref_id]
+    scoped_search relation: :policy, on: :name, rename: :policy_name
```

Comment:
I'm not sure how introducing a new search param "fixes" a problem, the search on the `name` key should search in the name of the policy instead.

Reply:
Should the name key only search through policy names? If so, I could just make the name search
`scoped_search relation: :policy, on: :name` but this wouldn't search canonical profiles.

Reply:
Well, you should to fix both issues then :grin: introducing a new query doesn't actually fix the existing one.

Reply:
Okay, the ext_method that I was working on was searching all profiles and policy names and this would be used for the `name` scoped search so I think this should solve the issue